stacks_queues/stack.py

```diff
@@ -21,18 +23,27 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store == None:
```

Comment:
`self.store` is initialized to a Linked List per the constructor so it shouldn't be `None` but rather an empty Linked List. We can use `empty` instead.
```suggestion
        if self.store.empty()
```